src/components/BundleFollowerToastMessage.tsx

```diff
@@ -0,0 +1,45 @@
+import { toast } from "react-toastify";
+import { ClipboardDocumentIcon } from "@heroicons/react/20/solid";
+import { XMarkIcon } from "@heroicons/react/20/solid";
+
+export const BundleFollowerToastMessage = ({
+  endoZipBase64Sha512,
+  closeToast = () => {},
+}: {
+  endoZipBase64Sha512: string;
+  closeToast: () => void;
+}) => (
+  <div className="flex items-start pointer-events-auto">
+    <div className="ml-3 w-0 flex-1 pt-0.5">
+      <p className="text-sm font-medium text-gray-900">
+        Bundle Successfully Installed!
+      </p>
+      <span className="mt-1 text-sm text-gray-500">
+        <span
+          className="text-sm text-blue-500 hover:text-blue-700 underline cursor-pointer"
+          onClick={async () => {
+            await window.navigator.clipboard.writeText(endoZipBase64Sha512);
```

Comment:
what would it cost to avoid ambient access to `window`?

Reply:
[c9c1537](https://github.com/DCFoundation/cosmos-proposal-builder/pull/39/commits/c9c1537aa6c26980325a6da0fd597b0ef00e4bad)